### ISSUE TITLE
[TASK] Replace daisyUI by something else

### ISSUE THREAD

**Issue by mohe2015** [reform-org/reform#297]
Estimated story points:

# Description
 
It has lot of accessibility issues and lots of missing features

 

# Acceptance criteria

**Comment by lukasschreiber** [comment]
actually we don't really use any feature except drawer and dropdown functionality, everything else could be easily replaced
